changelog.d/14525.feature

```diff
@@ -0,0 +1 @@
+Stop using deprecated `keyIds` parameter when calling `/_matrix/key/v2/server`.
```

Comment:
Maybe combine the changelog for #14490 into this one?

Reply:
done